schemas/webpackOptionsSchema.json

```diff
@@ -823,7 +823,8 @@
       "$ref": "#/definitions/entry"
     },
     "externals": {
-      "$ref": "#/definitions/externals"
+      "type": "object",
```

Comment:
``` js
externals: /xyz/
externals: ["a", "b"]
externals: "xyz"
```

These are also valid options.

Reply:
Ah ok. I'll correct that

Reply:
Actually, what does the second two mean? I don't see those forms documented or tested:

```
externals: ["a", "b"]
externals: "xyz"
```

Especially with the array, its ambiguous to me whether its defining that both "a" and "b" are external modules, or whether it means a sub property corresponding to `root["a"]["b"]`.

Reply:
Ok no I got it. `["a","b"]` is defining both of those as external modules. Will fix it.